app/assets/stylesheets/incidents.scss

```diff
@@ -71,3 +71,8 @@ table.history>tbody>tr{
     }
   }
 }
+
+.high-contrast-options option{
+  color: black;
+  background: white;
```

Comment:
I'm a little confused about option styling in general. I thought how those appear is pretty browser-dependent.